cobra_test.go

```diff
@@ -42,9 +42,10 @@ var cmdEcho = &Command{
 }
 
 var cmdTimes = &Command{
-	Use:   "times [# times] [string to echo]",
-	Short: "Echo anything to the screen more times",
-	Long:  `a slightly useless command for testing.`,
+	Use:        "times [# times] [string to echo]",
+	Short:      "Echo anything to the screen more times",
+	Long:       `a slightly useless command for testing.`,
+	Deprecated: true,
```

Comment:
The difference between deprecation and use is that the deprecation warning
should actually print whenever you use a deprecated feature.

On Tue, Apr 7, 2015 at 3:10 PM, Eric Paris <notifications@github.com> wrote:

> In cobra_test.go
> <https://github.com/spf13/cobra/pull/90#discussion_r27926160>:
>
> > @@ -42,9 +42,10 @@ var cmdEcho = &Command{
> >  }
> >
> >  var cmdTimes = &Command{
> > -	Use:   "times [# times] [string to echo]",
> > -	Short: "Echo anything to the screen more times",
> > -	Long:  `a slightly useless command for testing.`,
> > +	Use:        "times [# times] [string to echo]",
> > +	Short:      "Echo anything to the screen more times",
> > +	Long:       `a slightly useless command for testing.`,
> > +	Deprecated: true,
>
> And use that in place of Use (or short or long)? Should projects just
> change "Use" and we just not support 'deprecation' ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spf13/cobra/pull/90/files#r27926160>.
>